src/client/sound/sound.c

```diff
@@ -205,17 +205,63 @@ S_IsSilencedMuzzleFlash(const wavinfo_t* info, const void* raw_data, const char*
 	return true;
 }
 
+static void
+S_LoadVorbis(char *path, char* name, wavinfo_t *info, void **buffer)
+{
+	int	len;
+	char namewe[256];
+	char filename[MAX_QPATH];
+	const char* ext;
+
+	if (!path)
+	{
+		return;
+	}
+
+	ext = COM_FileExtension(path);
+	if(!ext[0])
+	{
+		/* file has no extension */
+		return;
+	}
+
+	len = strlen(path);
+
+	/* Remove the extension */
+	memset(namewe, 0, 256);
+	memcpy(namewe, path, len - (strlen(ext) + 1));
+
+	if (len < 5)
+	{
+		return;
+	}
+
+	if (strcmp(ext, "wav"))
```

Comment:
what's happening here?

Reply:
It tried to check that originally file was wav, i have removed such check in 3cc39bb48e3b42847e6e5f6024e7c460f83ce114 as redundant.